website_snippet_contact_form/__openerp__.py

```diff
@@ -17,5 +17,6 @@
     "data": [
         "views/snippets.xml",
         "views/assets.xml",
+        "data/website_snippet_contact_form.xml",
```

Comment:
This should get moved to `views/pages.xml` or something with a more descriptive name under views folder.